packages/utils/src/internals/sitemap.ts

```diff
@@ -223,49 +234,75 @@ export async function* parseSitemap<T extends ParseSitemapOptions>(
         if (source.type === 'url') {
             const sitemapUrl = new URL(source.url);
             visitedSitemapUrls.add(sitemapUrl.toString());
+            let retriesLeft = sitemapRetries;
+
+            while (--retriesLeft > 0) {
+                try {
+                    const sitemapStream = await new Promise<ReturnType<typeof gotScraping.stream>>(
+                        (resolve, reject) => {
+                            const request = gotScraping.stream({
+                                url: sitemapUrl,
+                                proxyUrl,
+                                method: 'GET',
+                                timeout: networkTimeouts,
+                                headers: {
+                                    'accept': 'application/xhtml+xml,application/xml,text/plain',
```

Comment:
Can't this make a well-behaved server not return a .gz file? I'd say Content negotiation failed in this case.

Reply:
This is (a shortened) version of what regular web browsers send in the `accept` header. I added a `*/*` directive to allow for any type the server has. The `accept-encoding` header is something `got-scraping` sets itself, as it depends on the HTTP client capabilities.

I say let's monitor for errors like this, worst case, we know where to look :)

Reply:
Do browsers send that for files that end in .gz as well? It sounds fishy...